sdk/containerinstance/Microsoft.Azure.Management.ContainerInstance/src/Generated/ContainersOperations.cs

```diff
@@ -91,7 +95,7 @@ internal ContainersOperations(ContainerInstanceManagementClient client)
         /// <return>
         /// A response object containing the response body and response headers.
         /// </return>
-        public async Task<AzureOperationResponse<Logs>> ListLogsWithHttpMessagesAsync(string resourceGroupName, string containerGroupName, string containerName, int? tail = default(int?), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task<AzureOperationResponse<Logs>> ListLogsWithHttpMessagesAsync(string resourceGroupName, string containerGroupName, string containerName, int? tail = default(int?), bool? timestamps = default(bool?), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
This seems like a breaking change: introducing `timestamps` optional param not at the end.

Please rearrange the params in the swagger file so `bool? timestamps` is at the end.

Reply:
@JonathanCrd The timestamps param is already at the end of the param list in the Swagger for the Containers_ListLogs operation (https://github.com/Azure/azure-rest-api-specs/blob/master/specification/containerinstance/resource-manager/Microsoft.ContainerInstance/stable/2021-03-01/containerInstance.json)

Could this be an issue with the autogen? How could we fix it?

Reply:
@JonathanCrd Following up here. Since the parameter in the Swagger is already in the correct order, what can I do to correct this? Since this is generated by the autogen, I assume I cannot manually change the order.

Reply:
Hi, I'm asking how to resolve this, since this has to do with the autogen. 
I'll let you know what to do, or I'll approve the PR if there's nothing else for you to do.

Reply:
@joseph-porter, this issue is specific to .NET generation, and the way to fix it is by adding a partial class with an overload that implements the old API, generally this is done by making all the parameters up to the last common parameter required.

For example, in this case it would be: 
```C#
public async Task<AzureOperationResponse<Logs>> ListLogsWithHttpMessagesAsync(string resourceGroupName, string containerGroupName, string containerName, int tail, Dictionary<string, List<string>> customHeaders, CancellationToken cancellationToken = default(CancellationToken))
```

In that way, any calls that provided a value form customHeaders will use this added method, but any new method calls, and any method calls that do not use customer headers will automatically use the new method.

Thanks to @markcowl for providing this solution :)

Reply:
@JonathanCrd  I have added an overloaded method with the old API.